recommendation/ionic.bzip2.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Name": "Ionic.BZip2",
+  "Version": "1.0.0",
+  "Packages": [
+    {
+      "Value": "Ionic.Zip",
+      "Type": "Nuget"
+    }
+  ],
+  "Recommendations": [
+    {
+      "Type": "Namespace",
+      "Name": "Ionic.BZip2",
+      "Value": "Ionic.BZip2",
+      "KeyType": "Name",
+      "ContainingType": "",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [
+                "x86",
+                "x64",
+                "ARM32",
+                "ARM64"
+              ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [
+                "x86",
+                "x64",
+                "ARM32",
+                "ARM64"
+              ]
+            }
+          ],
+          "Description": "Add a reference to DotNetZip",
```

Comment:
For consistency, add a "." after each description

Reply:
Good catch, will fix